### DISCUSSION TITLE
Google Api Error: badRequest: Changes cannot be sent for review automatically

### DISCUSSION THREAD

**Discussion by Thangaraj-M** [fastlane/fastlane#18935] [Ideas]
When upload the apk for an app to google play with Supply lane, getting the below error when that app was rejected from google review previously.
**Google Api Error: badRequest: Changes cannot be sent for review automatically. Please set the query parameter changesNotSentForReview to true. Once committed, the changes in this edit can be sent for review from the Google Play Console UI.**

Currently there is no option in Supply lane to set the changesNotSentForReview to true. 

The Google API for commit the changes is having this param. https://developers.google.com/android-publisher/api-ref/rest/v3/edits/commit.

Now this param need to be added to Supply lane and that value should be passed to the Google API when commit the changes.

**Comment by Sanfored** [comment]
Google Api Error: badRequest: Changes cannot be sent for review automatically. Please set the query parameter changes Not Sent For Review to true. Once committed, the changes in this edit can be sent for review from the Google Play Console UI. - Changes cannot be sent for review automatically. Please set the query parameter changes Not sent For Review to true. Once committed, the changes in this edit can be sent for review from the Google Play Console UI.

[firstcallonline login](https://www.firstcallonline.us/)

**Reply by Panajev** [reply]
This is something that has bitten us too :/...
